src/mock-dnas/mockData.js

```diff
@@ -93,14 +93,13 @@ const data = {
   },
   hha: {
     host: {
-      register_as_host: args => {
-        console.log(" `host_doc` args passed when registering host; reqs for host registaration are currently being disregarded.) : ", args)
-        
-        data.hha.host.is_registered_as_host = { result: 'QmHHAHostRegistrationAddress' }
-        return data.hha.host.is_registered_as_host
+      register_as_host: () => {
```

Comment:
this has been refactored so that it behaves like the dna (as far as I can tell. Haven't read the dna code, just fired it up and poked it a bit to see what it did).
let me know if you see anything I've missed or misunderstood here.